tv.sh

```diff
@@ -8,34 +8,38 @@ TITLES=(
   "SIC"
   "TVI"
   "RTP 3"
-  "SIC Noticias"
+  "SIC Notícias"
   "CNN Portugal"
-  "RTP Memoria"
+  "RTP Memória"
   "RTP Madeira"
-  "RTP Acores"
-  "RTP Africa"
+  "RTP Açores"
+  "RTP África"
   "RTP Internacional"
-  "SIC Internacional"
+  "TVI Internacional"
+  "TVI Ficção"
+  "TVI Reality"
   "ARTV"
   "Porto Canal"
   "Euronews"
   "Kuriakos TV"
 )
 
 STREAMS=(
-  "__rtp rtp1"
-  "__rtp rtp2"
-  "https://sic.live.impresa.pt/sic.m3u8"
+  "https://streaming-live-app.rtp.pt/liverepeater/smil:rtp1HD.smil/rtp_playlist.m3u8"
+  "https://streaming-live-app.rtp.pt/liverepeater/smil:rtp2HD.smil/rtp_playlist.m3u8"
+  "https://d1zx6l1dn8vaj5.cloudfront.net/out/v1/b89cc37caa6d418eb423cf092a2ef970/index.m3u8"
   "__tvi"
-  "__rtp rtp3"
+  "https://streaming-live-app.rtp.pt/liverepeater/smil:rtpnHD.smil/rtp_playlist.m3u8"
   "https://sicnot.live.impresa.pt/sicnot.m3u8"
-  "__tvi24"
+  "__cnnpt"
   "__rtp rtpmemoria"
   "__rtp rtpmadeira"
   "__rtp rtpacores"
   "__rtp rtpafrica"
   "__rtp rtpinternacional"
-  "http://live.impresa.pt/live/sicint/sicint.m3u8"
+  "__tviint"
+  "https://video-auth2.iol.pt/live_tvi_ficcao/live_tvi_ficcao/edge_servers/tvificcao-720p/playlist.m3u8"
```

Comment:
These two channels are paid (when accessed though the TVI website).
I have a feeling that they will eventually stop working.

Reply:
They have been working for years now. I have them on the [M3UPT](https://github.com/LITUATUI/M3UPT) list.